SpatialGDK/Extras/schema/query_tags.schema

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+package unreal;
+
+// System query tags for entity completeness
+
+component ServerAuthTag {
+    id = 1000;
```

Comment:
The user defined ops are in the range [1000, 1999] I think. I'm not certain if 2000 is valid or not. Right now the spatialdispatcher will invoke the seldom used user callback thing for this though.

Should probably start these at 2001 and go up from there. Or we place them below that and above the improbable standard library. Although keeping gdk standard tags in the gdk standard component range is probably good.

Reply:
changing to be 2001 up